### PR TITLE
Story/11827 changes recovery certificates details

### PR DESCRIPTION
**Pull request by naveeddotio**
## Description
Changes related to recovery certificates on various screens

## Link to Jira
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-11827

## Screenshots
WhatsApp Image 2022-03-09 at 13 19 33-2
WhatsApp Image 2022-03-09 at 13 19 34
WhatsApp Image 2022-03-09 at 13 19 34-2
WhatsApp Image 2022-03-09 at 13 19 33
WhatsApp Image 2022-03-09 at 13 19 31
WhatsApp Image 2022-03-09 at 13 19 32

### CODE REVIEW

**Review by CV113 — APPROVED** [review]
UA approved

File: src/xcode/ENA/ENA/Source/Services/HealthCertificate/Models/Extensions/RecoveryEntry+Extension.swift [hunk 1]
```diff
@@ -15,6 +15,10 @@ extension RecoveryEntry {
 		return ISO8601DateFormatter.justLocalDateFormatter.date(from: certificateValidUntil)
```
Comment (ArturFriesen) [inline src/xcode/ENA/ENA/Source/Services/HealthCertificate/Models/Extensions/RecoveryEntry+Extension.swift:15]: Can 'localCertificateValidityStartDate' and 'localCertificateValidityEndDate' be removed from the extension?

**Review by ArturFriesen — CHANGES_REQUESTED** [review]

**Review by luisoos — APPROVED** [review]

**Review by ArturFriesen — APPROVED** [review]

**Comment by Ein-Tim** [comment]
@naveeddotio I have a question on the last picture you used in this PR. Is this actually going to happen? The text talks about vaccination certificates, however, the recovery certificate is the certificate to be updated. 

If this should not happen in productive environment: Ok, no problem.
If this is actually possible to happen in productive environment: I suggest to change the text.

Ping @thomasaugsten

**Comment by thomasaugsten** [comment]
The last screenshot cannot happen at the moment in production. Because only 2/2 vaccination certificate can be replaced at the moment.

**Pull request merged**